node/node.go

```diff
@@ -287,6 +296,14 @@ func New(host p2p.Host, consensus *bft.Consensus, db ethdb.Database) *Node {
 		}
 		node.Consensus.ConsensusBlock = make(chan *bft.BFTBlockInfo)
 		node.Consensus.VerifiedNewBlock = make(chan *types.Block)
+
+		var err error
+		node.Pubsub, err = pubsub.NewFloodSub(context.Background(), node.host.GetP2PHost())
```

Comment:
pubsub - is it what eugene is working on now?

Reply:
yes, this is a placeholder.